packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/components/SGVGridCell/SGVGridCell.js

```diff
@@ -79,14 +104,31 @@ function SGVGridCell (props) {
           transform={`scale(${zoom}) translate(${panX}, ${panY})`}
           transform-origin={`${imageWidth/2}px ${imageHeight/2}px`}
         />
+        <rect
+          fill="none"
+          stroke={(cellAnnotated) ? cellStyle.highlight : cellStyle.stroke}
+          strokeWidth={(cellAnnotated)
+            ? cellStyle.highlightWidth * BORDER_MULTIPLIER
+            : cellStyle.strokeWidth * BORDER_MULTIPLIER
+          }
+          width={cellWidth}
+          height={cellHeight}
+        />
+        {annotationMode  && (
+          <ClickableRect
+            tabIndex={0}
```

Comment:
```suggestion
            tabIndex={0}
            role='checkbox'
            aria-checked={checked}
            aria-label={`Subject ${index}`}
```
I think the subject ID might be a better label here but array index will work too.